content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,64 @@
+## Installation
+
+- **Added** pod anti-affinity and toleration support to all of our control plane components.
+- **Added** `sidecarInjectorWebhook.neverInjectSelector` and `sidecarInjectorWebhook.alwaysInjectSelector` to allow users to further refine workloads that have sidecar automatically injected or not, based on label selectors.
+- **Added** `global.logging.level` and `global.proxy.logLevel` to allow users to easily configure logs for control plane and data plane components globally.
+- **Added** support to configure the Datadog location via `global.tracer.datadog.address`.
+
+## Upgrades
+
+- **Removed** deprecated Adapter and Template CRDs by default. Configure `mixer.templates.useTemplateCRDs=true` and `mixer.adapters.useAdapterCRDs=true` to re-enable them.
+
+Refer to the [installation option change page](/docs/reference/config/installation-options-changes/) to view the complete list of changes.
+
+## General
+
+- **Added** `traffic.sidecar.istio.io/includeInboundPorts` annotation to eliminate the need for service owner to declare `containerPort` in the deployment yaml file.
+- **Introduced** IPv6 alpha support for Kubernetes clusters.
+
+## `istioctl` and `kubectl`
+
+- **Graduated** `istioctl verify-install` out of experimental.
+- **Improved** `istioctl verify-install` to validate if a given Kubernetes environment meets Istio's prerequisites.
+- **Added** auto-completion support to `istioctl`.
+- **Added** `istioctl experimental dashboard` to allow users to easily open the web UI of any Istio addons.
+- **Added** `istioctl x` alias to conveniently run `istioctl experimental` command.
+- **Improved** `istioctl version` to report both Istio control plane and `istioctl` version info by default.
+- **Improved** `istioctl validate` to validate Mixer configuration and supports deep validation with referential integrity.
+
+## Traffic Management
+
+- **Improved** locality based routing in multicluster environments.
+- **Improved** outbound traffic policy in `ALLOW_ANY` mode. Traffic for unknown HTTP/HTTPS hosts on an existing port will be forwarded as is. Unknown traffic will be logged in Envoy access logs.
+- **Added** support for sending traces to Datadog.
+- **Added** support for setting HTTP idle timeouts to upstream services.
+- **Improved** Sidecar support for NONE mode (without iptables).
+- **Added** support for configuring the paths for Istio mutual TLS certificates.
+- **Added** ability to configure the DNS refresh rate for sidecar Envoys, to reduce the load on the DNS servers.
+
+## Security
+
+- **Added** Kubernetes health check prober rewrite per deployment via `sidecar.istio.io/rewriteAppHTTPProbers: "true"` in the `PodSpec` annotation.
+- **Fixed** mutual TLS probe rewrite for HTTPS probes
+- **Added** Support for [PKCS 8](https://en.wikipedia.org/wiki/PKCS_8) private keys for workloads, enabled by the field `pkcs8-keys` on Citadel.
+- **Fixed** SAN field in workload certificates is set as `critical`.
```

Comment:
Needs a link. What does 'critical' mean?

Reply:
@wenchenglu

Reply:
Probably need to link to [RFC5280, section 4.2](https://tools.ietf.org/html/rfc5280#section-4.2)

> Each extension in a certificate is designated as either critical or non-critical.  A certificate-using system MUST reject the certificate if it encounters a critical extension it does not recognize or a critical extension that contains information that it cannot process.  A non-critical extension MAY be ignored if it is not recognized, but MUST be processed if it is recognized.

Reply:
someone updated!!